memgpt/cli/cli.py

```diff
@@ -466,6 +467,21 @@ def run(
             )
         except ValueError as e:
             typer.secho(f"Failed to create agent from provided information:\n{e}", fg=typer.colors.RED)
+            # Delete the directory of the failed agent
```

Comment:
@sarahwooders not sure if this is too dangerous (rm -r) - alternatively we could move to /tmp instead of deleting?